packages/components/src/components/ContentCard/ContentCard.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react";
+import { Box, BoxProps } from "../../primitives/Box";
+import { Heading } from "../Heading";
+import { Text } from "../../primitives/Text";
+import { Icon } from "../Icon";
+import { Button } from "../Button";
+import { Anchor } from "../Anchor";
+
+type ImagePosition = "right" | "top";
+
+type Background = "default" | "emphasized" | "inverse";
+
+type CommonProps = {
+  imageSrc: string;
+  imageAlt: string;
+  title: string;
+  text: string;
+  tag?: string;
+  date?: string;
+  buttonText?: string;
+  onClickButton?: () => void;
+  imagePosition?: ImagePosition;
+  iconUrl?: string;
+  background?: Background;
+};
+
+type InteractiveCard = {
+  href: string;
+  linkHref?: undefined;
+  linkText?: undefined;
+  as?: undefined;
+};
+
+type RegularCard = {
+  href?: undefined;
+  linkHref?: string;
```

Comment:
Isn't it the opposite? The Interactive have "clickable" links? 🤔

Reply:
I got that the other way. I'll double check before merging it